src/main/resources/sql

```diff
@@ -0,0 +1,226 @@
+-- Table: address
```

Comment:
KONIECZNIE rozbić ten plik na kilka.

1. Tworzenie struktury (tabele, constraints itp.)
2. Dane statyczne (predefiniowane role itp.)
3. Dane testowe - to nie powinno być ładowane na produkcji, a tylko do testów.